HW_3/task_3.py

```diff
@@ -0,0 +1,24 @@
+"""
+Задайте список из вещественных чисел, округленных до сотых.
+Найдите разницу между максимальным и минимальным значением дробной части элементов.
+
+Ввод: значение типа <list> (либо значения типа <int> – размерность списка)
+Вывод: значение типа <float>
+
+Пример:
+[1.1, 1.2, 3.1, 5, 10.01]
+2.0
+"""
+
+
+
+import random
+
+number = int(input("Задайте размерность списка: "))
+my_list = [round(random.uniform(1, 5), 2) for i in range(number)]
+list_2 = []
+for i in range(len(my_list)):
+    list_2.append(my_list[i] - int(my_list[i]))
```

Comment:
И вообще. Функцию `range` использовать совсем не обязательно. Можно пройтись прямо по элементам списка.
```suggestion
for elem in my_list:
    list_2.append(elem % 1)
```